utilities/console.py

```diff
@@ -108,10 +115,44 @@ def __enter__(self):
         """
         Connect to console
         """
-        return self.connect()
+        self.connect()
+        return self
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         """
         Logout from shell
         """
         self.disconnect()
+
+    def run_commands(
+        self,
+        commands: list[str],
+        timeout: int = TIMEOUT_1MIN,
+        verify_commands_output: bool = True,
+        command_output: bool = False,
+    ) -> dict[str, list[str]] | None:
+        """
+        Run a list of commands inside VM and (if verify_commands_output) check all commands return 0.
+        If return code other than 0 then it will break execution and raise exception.
+
+        Args:
+            commands (list): List of commands
+            timeout (int): Time to wait for the command output
+            verify_commands_output (bool): Check commands return 0
+            command_output (bool): If selected, returns a dict of command and associated output
+        """
+        output = {}
```

Comment:
Please add type to `output`

Reply:
Changed, waiting to push after the other points have been clarified.